modules/tile-converter/src/types.ts

```diff
@@ -136,3 +168,200 @@ type FieldInfo = {
   isEditable: boolean;
   label: string;
 };
+
+// Spec - https://github.com/Esri/i3s-spec/blob/master/docs/1.7/3DNodeIndexDocument.cmn.md
+export type Node3DIndexDocument = {
```

Comment:
I would create a separate file `i3s-types.ts` that just describes the I3S spec (any other types would go into `types.ts`). There is a similar setup in `modules/gltf`.

That file should long-term be in `modules/i3s`.

Reply:
Will move in separate PR.